Library/Homebrew/test/release_notes_spec.rb

```diff
@@ -12,21 +12,27 @@
       system "git", "commit", "--allow-empty", "-m", "Merge pull request #1 from Homebrew/fix", "-m", "Do something"
       system "git", "commit", "--allow-empty", "-m", "make a change"
       system "git", "commit", "--allow-empty", "-m", "Merge pull request #2 from User/fix", "-m", "Do something else"
+      system "git", "commit", "--allow-empty", "-m", "another change"
+      system "git", "commit", "--allow-empty", "-m", "Merge pull request #3 from User/another_change"
     end
   end
 
   describe ".generate_release_notes" do
     it "generates release notes" do
+      # rubocop:disable Style/TrailingWhitespace
```

Comment:
Seems undesirable to have trailing whitespace generated here?

Reply:
that was a placeholder, and it's gone now